include/boost/histogram/accumulators/efficiency.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_ACCUMULATORS_EFFICIENCY_HPP
+#define BOOST_HISTOGRAM_ACCUMULATORS_EFFICIENCY_HPP
+
+#include <boost/core/nvp.hpp>
+#include <boost/histogram/fwd.hpp> // for efficiency<>
+#include <boost/histogram/detail/probit.hpp>
+#include <boost/histogram/detail/waldInterval.hpp>
+#include <boost/histogram/detail/wilsonInterval.hpp>
+#include <utility>
+#include <limits>
+
+enum class interval_type {
+    wald,
+    wilson,
+    jeffreys,
+    agrestiCoull,
+    clopperPearson
+};
+
+struct efficiency {
```

Comment:
You describe efficiency in fwd.hpp as a templated type, but this type is not templated. We eventually want to make efficiency templated on the type of the counters, right now it is hard-coded to double. We want to eventually make efficiency templated, but I think you should first write some unit tests and then template this class once the unit tests are in place.